netdata.spec.in

```diff
@@ -1,90 +1,100 @@
+#
+# Conditional build:
+%bcond_without	systemd		# systemd
+%bcond_without	nfacct		# build with nfacct plugin
```

Comment:
but we need this to be on by default in fedora, without any extra parameter to rpmbuild. as far as I understand this will be off by default.